### DISCUSSION TITLE
My refresh rate is only 60 instead of 360

### DISCUSSION THREAD

**Discussion by ChaerilM** [seerge/g-helper#3239] [Q&A] [answered]
help, my monitor now run 60 + OD instead of 360 +OD. 360 option is gone from windows settings as well. 
what should i do.

![image](https://github.com/user-attachments/assets/f8e55f0b-b0af-453c-b18d-3f16d530ae92)

**Comment by seerge** [comment, marked as answer]
@ChaerilM hello, if it's gone in windows - then it's either not supported (no idea what laptop you have) or something is wrong with [GPU drivers that you may want to reinstall](https://github.com/seerge/g-helper/wiki/Troubleshooting#reinstalling-gpu-drivers)

**Reply by ChaerilM** [reply]
yes, seems ddu is the answer. i think the problem comes from the new all in one nvidia app. but i have not been able to reinstall it again.
